docs: Add requirement for bitcoind. (#8119)
lightningnetwork/lnd: docs/INSTALL.md

  to provide a reliable delivery of notifications (e.g.
  `--zmqpubrawblock=tcp://127.0.0.1:28332` and
  `--zmqpubrawtx=tcp://127.0.0.1:28333`).

> - Make sure you do NOT use the bitcoind config setting `--rpcserialversion`.

Reviewer: ```suggestion
- Make sure the config `--rpcserialversion` in `bitcoind` is NOT used. 
```

Reviewer 2: Ah, I missed this before, but the config flags in `bitcoind` only have a single dash.

>   Lightning depends on segwit transactions and the default rpc bitcoind

Reviewer: We should mention this `-rpcserialversion` must be set to 1 and if we don't explicitly set it in the config file we are fine as it's already default to 1.

Also feel like there's no need to explain in details other than when setting config value to 0 the witness data won't be returned.

  behaviour includes the witness data. Changes to this setting are safety
  critical and might lead to loss of funds.

>   

Reviewer: Maybe we can also link the discussion here?
https://github.com/bitcoin/bitcoin/issues/28730
